stdlib/public/Cxx/std/String.swift

```diff
@@ -95,6 +95,30 @@ extension std.u32string {
   }
 }
 
+extension std.wstring {
```

Comment:
I wonder if it would make sense auto conforming `wstring` to the corresponding `u*string` instead. Aren't they supposed to desugar to the same type?

Reply:
Nope, they are distinct types! `wchar_t` isn't the same type as `char16_t` or `char32_t`, even if the bitwidth matches.